### ISSUE TITLE
radar data to apollo

### ISSUE THREAD

**Issue by WilliaJing** [ApolloAuto/apollo#14960]
Hi,Does anyone know?
I want to know, I send RADAR data to Apollo by simulating simulation, and how to verify on the Apollo side to verify that the data is correct or not.

**Comment by daohu527** [comment]
1. You should first follow the radar data format
https://github.com/ApolloAuto/apollo/blob/master/modules/common_msgs/sensor_msgs/conti_radar.proto

2. Then use `cyber_monitor` to find radar topics and data is right

**Comment by WilliaJing** [comment]
@daohu527 I know this,Can I see the effect in DreamView?Can Radar data make the perception obstacles more accurate?thanks your reply.

**Comment by daohu527** [comment]
Yes, the radar will be fused with the lidar and camera, and the fusion result will be output.

You can also just output rader results, but we need to modify the output topic to `/apollo/perception/obstacles` and message to `PerceptionObstacles`
```
radar SensorFrameMessage  ---\
lidar SensorFrameMessage  --- >  fusion ---> PerceptionObstacles
camera SensorFrameMessage ---/

// change to
radar  --->  PerceptionObstacles
```
## message introduction
The fusion process will use the internal data structure `SensorFrameMessage`,which invisible to others, and then output the final external data structure `PerceptionObstacles`.

Dreamview will just display `PerceptionObstacles` but not display the inner messages, if we want to separately display radar data, we should just start radar component and modify the output message to `PerceptionObstacles`

**Comment by WilliaJing** [comment]
>  we should just start radar component and modify the output message to `PerceptionObstacles`

thanks;as I know, Radar component in dreamview is used to control hardware,but I am simulation radar data in CARLA,
now i am writing to channel '/apollo/perception/obstacles',Is it not necessary to set up radar sensors and write channel '/apollo/sensor/radar/front'?because radars are used to perceive obstacles.
